### PR TITLE
Fixup Eurobraille docs

### PR DESCRIPTION
**Pull request by seanbudd**
### Link to issue number:
Follow up to #14690 

### Summary of the issue:
The docs for the eurobraille display aren't entirely clear and could use some rewording

### Description of user facing changes
Fix up docs

### Description of development approach
Fix up docs

### Testing strategy:
N/a
### Known issues with pull request:
n/a
### Change log entries:
New features
Changes
Bug fixes
For Developers

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

### CODE REVIEW

**Comment by seanbudd** [comment]
cc @FalkoBabbage - can you please review these changes?

**Comment by FalkoBabbage** [comment]
I have passed this on to Eurobraille since they provided the documentation.  I'll call them today as well.

**Comment by FalkoBabbage** [comment]
Eurobraille agrees with the changes. 
Below is the answer I got:
> The changes are consistent with the documentation I sent you.
>
> 
> To add clarity to this documentation regarding key assignments we can add :
>
> ------------------------------
> 
> The Eurobraille driver is a generic driver for all its devices.
> 
> It is for this is why the driver key mapping refers keys L1 to L8 of Esytime.
> For Esys and b.note these keys does not exists.
> For b.book : L1 is backward key, L8 is forward key and L2 to L7 are the keys named C1 to C6.
> 
> For the same reason the driver key mapping refers Joystick1 and Joystick2 but on b.note these joysticks are 2 keypads of 4 keys.
>
> ------------------------------
> 
> I think a keyboard description for each braille devices is not necessary in NVDA documentation because users have a complete description in device documentation, but if it seems necessary to you or NVDA teams I can send you a short face description for each display ?

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -3659,13 +3659,13 @@ Due to this, and to maintain compatibility with other screen readers in Taiwan,
 
 ++ Eurobraille displays ++[Eurobraille]
 The b.book, b.note, Esys, Esytime and Iris displays from Eurobraille are supported by NVDA.  
-These devices have a braille keyboard with 10 keys. 
+These devices have a Braille keyboard with 10 keys.
```
Comment (Qchristensen) [inline user_docs/en/userGuide.t2t:3662]: The word "Braille" only needs to be capitalised when referring to the inventor of the reading system.  Referring to the code itself, lowercase is usual (as is done in the rest of this PR).  Ref: https://www.brailleauthority.org/capitalization/capitalization.html

**Review by Qchristensen — CHANGES_REQUESTED** [review]
All looks fine (with the note on capitalisation of braille)

**Comment by seanbudd** [comment]
Thanks @FalkoBabbage 

Further information isn't necessary, but we would welcome any extra documentation which helps:
- a user use the braille device with NVDA
- a developer understand the braille driver and how it talks to the braille device

**Pull request merged**